CITATION.cff

```diff
@@ -56,7 +56,7 @@ keywords:
   - Existential Rules
 license: Apache-2.0
 commit: 6ccd3fcc12d57fbac5e3d7e5c5547e72a610d7f7
-version: 0.5.1
+version: 0.6.0
```

Comment:
If you bump the version, you also need to bump the commit, 6ccd3fcc12d57fbac5e3d7e5c5547e72a610d7f7 corresponds to the v0.5.1 tag.

Reply:
Good point